MonsterWaves/Screen/BaseScreen.hpp

```diff
@@ -0,0 +1,16 @@
+#include "Window.hpp"
+
+class BaseScreen
+{
+public:
+	virtual BaseScreen() = default;
+	virtual ~BaseScreen() = default;
+protected:
+	virtual void update() = 0;
+	virtual void render() = 0;
+	virtual void handleEvent() = 0;
+	virtual bool windowIsDone() = 0;
+private:
+	Window m_window;
```

Comment:
@JakubMastalski Dlaczego to jest private? Do tego muszą mieć dostęp klasy, które będą dziedziczyć...

Reply:
kurwa myslalem o tym nie wiem jak to sie stalo hahaha